### DISCUSSION TITLE
Multi-mappers, QTI-seq quality data and alternate start codons

### DISCUSSION THREAD

**Discussion by MMaqueda** [zhpn1024/ribotish#22] [General]
Hi,

   I have been using RiboTISH with QTI-seq public data (SRR1630828 and SRR1630830, also analyzed in RiboTISH paper) and I am a bit confused about the topics I mentioned in the subject. Hope you can shed some light here, specifically:

1) Is RiboTISH discarding the multi-mapper reads? 
I find it confusing because in the related paper, according to methods, the parameter -outSAMmultNmax is set to 1 (so allow for multimapping, just one hit) but then in the RPF length distribution, only those RPFs uniquely mapped to annotated protein- coding genes are considered.  I have obtained a huge number of multi-mapping reads and was wondering if they are really 'useless' or not.

2) Based on your experience about the TIS enrichment score, is there any minimum score value that we could consider to assume acceptable quality data?
Since it is QTI-seq data, we were expecting to have most of the reads accumulated around the translational start regions. However, we have seen multiple reads aligned to other gene exons. We would assume this profile with Ribo-seq data but not for QTI-seq.

3) Inspecting the results (without applying FDR q-value threshold),  we have seen that TIS site with ATG codon is not listed or not significant in some genes (i.e. for TP53 gene) while alternate codons are significantly predicted. Honestly, we expected to obtain TISs with ATG for all genes (with data) and additionally, zero or more alternate codons, are we wrong? I mean, is it right to have only significant TIS with alternate start codons but not with the canonical? 

And some minor question:

4) Just to confirm,  if –allresults are retrieved, the fisher’s FDR q-value threshold is not applied. Nevertheless, the Fisher’s pvalue default threshold (0.05) is applied, right? 
 
Many thanks in advance for any comment about this!!

Kind Regards, 

Maria Maqueda

**Comment by zhpn1024** [comment]
Hi Maria,

1. Ribo-TISH quality module uses only unique mapped reads, while predict module has an option --maxNH (default 5) to discard the reads with number of hits higher than the value.
2. We do not have suggested cutoff for the enrichment score. The TI-seq data are usually not as ideal as expected.
3. You are right. However, there may be some genes that do not have sufficient read counts in the expected ATG TISs in the TI-seq data. It may reflect TI-Seq limitations or fluctuations.
4. Yes. You can change the fspth.

Kind Regards,
Peng

**Reply by MMaqueda** [reply]
Hi Peng,

  Many thanks for your quick response. 

I was not aware of the --maxNH option, I will give it a try a tune it to assess the impact on the results.

OK for the rest of answers. 

King regards,

Maria

**Reply by zhpn1024** [reply]
I forgot to add descriptions of some options like --maxNH in README. I'll add them later.

**Comment by zhpn1024** [comment]
I forgot to add descriptions of some options like --maxNH in README. I'll add them later.

**Reply by MMaqueda** [reply]
Well, I guess you highlighted the most relevant to run the tool. After you mentioned it, I saw it with the predict -h option.

**Reply by zhpn1024** [reply]
Yes, all options are shown there.
